Assets/Scripts/DrawWalls.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class DrawWalls : MonoBehaviour {
+    private Transform leftWall;
+    private Transform rightWall;
+
+    void Start() {
+        leftWall = transform.Find("LeftWall").gameObject.transform;
+        rightWall = transform.Find("RightWall").gameObject.transform;
+    }
+
+	void Update() {
```

Comment:
I don't understand why github is displayed the spacing like this. It looks fine in Visual Studio -_-